docs/source/configuration/locking.md

```diff
@@ -0,0 +1,21 @@
+# Locking support
+
+Volto provides support for Plone's locking feature, but you need to enable the `plone.locking` behavior on your Dexterity content types first. You can do that in Plone's control panel or using the Generic Setup facilities.
```

Comment:
Recommendation: I usually start a new section by explaining the Plone feature in 2-3 sentences just in case the audience might not be familiar with the feature.

Reply:
@tisto Updated.